src/Cli/ConfigGenerator.cs

```diff
@@ -698,13 +698,30 @@ private static bool TryConfigureRuntimeOptions(
             ConfigureOptions options,
             [NotNullWhen(true)] ref RuntimeConfig runtimeConfig)
         {
+            // Rest: Enabled, Path, and Request.Body.Strict
+            if (options.RuntimeRestEnabled != null ||
+                options.RuntimeRestPath != null ||
+                options.RuntimeRestRequestBodyStrict != null)
+            {
+                RestRuntimeOptions? updatedRestOptions = runtimeConfig?.Runtime?.Rest ?? new();
+                bool status = TryUpdateConfigureRestValues(options, ref updatedRestOptions);
+                if (status)
+                {
+                    runtimeConfig = runtimeConfig! with { Runtime = runtimeConfig.Runtime! with { Rest = updatedRestOptions } };
```

Comment:
The first one is to make sure that even if the runtimeOptions are null, an update should create that object a nd update the value which needs to be updated

Reply:
I don't think the nullable operator on `runtimeConfig?` is needed -> 
```csharp
RestRuntimeOptions? updatedRestOptions = runtimeConfig?.Runtime?.Rest ?? new();
RestRuntimeOptions? updatedRestOptions = runtimeConfig.Runtime?.Rest ?? new(); // ? is not needed since caller must provide runtimeConfig.
```

As a result, you no longer need the nullforgiving operator (!) here:
```csharp
runtimeConfig = runtimeConfig! with { Runtime = runtimeConfig.Runtime! with { Rest = updatedRestOptions } };
runtimeConfig = runtimeConfig with { Runtime = runtimeConfig.Runtime! with { Rest = updatedRestOptions } }; // no ! because you know runtimeConfig isn't null.

```